website/src/components/pageStateSelectors/WasapPageStateSelector.tsx

```diff
@@ -161,6 +161,8 @@ function ManualAnalysisFilter({
     pageState: WasapFilter;
     setPageState: (newState: WasapFilter) => void;
 }) {
+    const enabledMutationTypes: MutationType[] =
+        pageState.sequenceType === 'nucleotide' ? ['nucleotideMutations'] : ['aminoAcidMutations'];
```

Comment:
Then we could also use the constants here instead of "hardcoding" those strings.